src/asm/lexer.cpp

```diff
@@ -1173,6 +1171,10 @@ static Token readIdentifier(char firstChar, bool raw) {
 			return Token(search->second);
 	}
 
+	// Label scope `.` is the only nonlocal identifier that starts with a dot
+	if (identifier == ".")
```

Comment:
I would make that change if/when I make a PR for `..`.

Reply:
But doing this now would keep more error messages (and behaviour) consistent with when we do add `..` et al.

Reply:
I'll go ahead and add `..` to this PR before merge then.

Reply:
Actually no I won't, because we don't even track a "local scope" yet, so it would be a nontrivial change on top of this new feature. I'll just go ahead and change this then.

Reply:
Waiting for this to merge before implementing #1503.